app/models/order.py

```diff
@@ -7,7 +7,7 @@
 class OrderTicket(db.Model):
     __tablename__ = 'orders_tickets'
     order_id = db.Column(db.Integer, db.ForeignKey('orders.id', ondelete='CASCADE'), primary_key=True)
-    ticket_id = db.Column(db.Integer, db.ForeignKey('ticket.id', ondelete='CASCADE'), primary_key=True)
+    ticket_id = db.Column(db.Integer, db.ForeignKey('ticket.id', ondelete='CASCADE'), nullable=True)
```

Comment:
That was the issue,```CASCADE``` was trying to set ```None``` to ```ticket_id``` instead of deleting the row.

```Dependency rule tried to blank-out primary key column 'orders_tickets.ticket_id' on instance '<OrderTicket at 0x10c7debd0>'```

This was the error